tests/zfs-tests/tests/functional/slog/slog_replay_fs_001.ksh

```diff
@@ -145,15 +149,17 @@ log_must dd if=/dev/zero of=/$TESTPOOL/$TESTFS/holes.3 bs=128k count=2 \
    seek=2 conv=notrunc
 
 # TX_MKXATTR
-log_must mkdir /$TESTPOOL/$TESTFS/xattr.dir
-log_must attr -qs fileattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.dir
-log_must attr -qs tmpattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.dir
-log_must attr -qr tmpattr /$TESTPOOL/$TESTFS/xattr.dir
-
-log_must touch /$TESTPOOL/$TESTFS/xattr.file
-log_must attr -qs fileattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.file
-log_must attr -qs tmpattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.file
-log_must attr -qr tmpattr /$TESTPOOL/$TESTFS/xattr.file
+if is_linux; then
+	log_must mkdir /$TESTPOOL/$TESTFS/xattr.dir
+	log_must attr -qs fileattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.dir
+	log_must attr -qs tmpattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.dir
+	log_must attr -qr tmpattr /$TESTPOOL/$TESTFS/xattr.dir
+
+	log_must touch /$TESTPOOL/$TESTFS/xattr.file
+	log_must attr -qs fileattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.file
+	log_must attr -qs tmpattr -V HelloWorld /$TESTPOOL/$TESTFS/xattr.file
+	log_must attr -qr tmpattr /$TESTPOOL/$TESTFS/xattr.file
+fi
```

Comment:
Is there really no equivalent on FreeBSD even if different utilities are needed?

Reply:
Added the equivalent for FreeBSD.